### DISCUSSION TITLE
Project Board & Discord

### DISCUSSION THREAD

**Discussion by tj-kev** [flareact/flareact#150] [Ideas]
In #130 @subhendukundu mentioned the idea of a project board and discord channel to plan and discuss ideas for the framework. 

Would there be any interest in this from anyone? Do you think it would be helpful?

I think potentially the Discord first with the project board coming later once we have an idea of what a future roadmap may look like.

Suggestions welcome!

**Comment by traffisco** [comment]
I'm personally not a fan of real time messaging platforms for project  communications.
Discord make you scroll up endlessly to read what you have missed and usually full of off topic chatting.

**Reply by tj-kev** [reply]
That is a fair point.
